### PR TITLE
Added workaround for osx testing bug

### PR DESCRIPTION
**Pull request by SensibleSalmon**
see #1190 

@mr-c Fix done; merge?

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,7 @@
+2015-07-22  Jacob Fenton  <bocajnotnef@gmail.com>
+
+   * tests/test_read_parsers.py: added workaround for bug in OSX python
```
Comment (mr-c) [inline ChangeLog]: Python, the computer programming language,  is a proper noun :-)

**Comment by mr-c** [comment]
Checklist?

**Comment by bocajnotnef** [comment]
- [x] Is it mergeable?
- [x] Did it pass the tests?
- [x] If it introduces new functionality in scripts/ is it tested?
  Check for code coverage with `make clean diff-cover`
- [x] Is it well formatted? Look at `make pep8`, `make diff_pylint_report`,
  `make cppcheck`, and `make doc` output. Use `make format` and manual
  fixing as needed.
- [x] Did it change the command-line interface? Only additions are allowed
  without a major version increment. Changing file formats also requires a
  major version number increment.
- [x] Is it documented in the ChangeLog?
  http://en.wikipedia.org/wiki/Changelog#Format
- [x] Was a spellchecker run on the source code and documentation after
  changes were made?
- [x] Is the Copyright year up to date?

@mr-c Updated

**Comment by mr-c** [comment]
LGTM, thanks!

**Pull request merged**